### DISCUSSION TITLE
Stuck close channel

### DISCUSSION THREAD

**Discussion by AndrewAB86** [Ride-The-Lightning/RTL#1359] [Troubleshooting Questions]
Hey, while starting with my node i was playing around with channels. I closed a channel at set the fee to 1. It was stuck in waiting close for a long time. Then I had to replace the hard drive for the node, I restored the node and the channel close was gone but no funds. The channel close transaction is showing on my node again but not showing in close channels. How can I bump the fee to get this transaction finalised.

The transaction id is: b0f4c3968fc6b660b05a6562c349cfea6fd24babc055163f67041adac4d765f8

Help please.

Thanks

**Comment by saubyk** [comment]
Hi @AndrewAB86 is this a mutual close i.e. not a force close?

Also, what was the local balance on this channel? from the transaction it looks like it's going to one address so I am assuming it's all coming back to you.

So before we consider your options, closing a channel with 1 sat/vbyte is never a wise thing to do, unless you've looked at the mempool and you know it will confirm. And given that this transaction was broadcast 5 months ago, there was no chance of 1 sat/vbyte confirming. Hope you'll keep this in mind on your future lightning operations

Now, assuming all the balance on the channel is coming back to your node, you can use the bumpfee command on the cli to bump up the fee rate for this channel closing transaction. Keep in mind to use a much higher rate than the prevailing fee rates in the mempool, because this fee bumping will happen via cpfp, so the effective fee rate for your channel closing transaction will be less that what you specify when you run the bumpfee command.

Here's an example of the command syntax:
`lncli wallet bumpfee --sat_per_vbyte 50 <txid>:0`

**Reply by saubyk** [reply]
Also assuming that this is a LND node

**Reply by AndrewAB86** [reply]
Hi,

I now understand a bit more about fee's and realise that the fee I set was wrong and I always look at mempool before I do any transactions now. 

I have a lightning node using raspiblitz and the channel close was mutual. It has signed by both parties but hasn't closed because of the fee.

The balance is coming back to my node. Now I don't known much about terminal commands. If you could help with what I should be putting into the terminal to bump the fee would be great>

Thanks

**Reply by saubyk** [reply]
So, I gave you the exact syntax of the command you need to run
`lncli wallet bumpfee --sat_per_vbyte 50 <txid>:0`

If you need to figure out how to access the terminal on your Raspiblitz device to run the command, please reach out to the Raspiblitz support and they would guide you. Hope this helps

**Reply by AndrewAB86** [reply]
I can access the terminal, do i put the txid where the 0 is or in the brackets where it says txid

**Reply by saubyk** [reply]
where it says txid

**Reply by AndrewAB86** [reply]
great, I will give it a try. Thankyou

**Reply by AndrewAB86** [reply]
lncli wallet bumpfee --sat_per_vbyte 50 b0f4c3968fc6b660b05a6562c349cfea6fd24babc055163f67041adac4d765f8:0

I put this in the raspiblitz command and the response was as follows:
{

}
have i done this right?
Thanks

**Reply by AndrewAB86** [reply]
Also its not showing on my node again now

**Reply by AndrewAB86** [reply]
i tried it again in the terminal and this is the response I get: [lncli] rpc error: code = Unknown desc = the passed output does not belong to the wallet

**Comment by saubyk** [comment]
> lncli wallet bumpfee --sat_per_vbyte 50 b0f4c3968fc6b660b05a6562c349cfea6fd24babc055163f67041adac4d765f8:0
> 
> I put this in the raspiblitz command and the response was as follows: {
> 
> } have i done this right? Thanks

This usually means that transaction has been fee bumped. But I don't see anything in the mempool.

I would recommend increasing the size of the mempool on your node to 1000 and restart your node once. Before you attempt any other operation, inspect the transaction after restart in the mempool.

**Reply by AndrewAB86** [reply]
I have changed that and restarted the node. Mempool is still saying the same thing. The tx is not showing in my node anymore.

**Reply by AndrewAB86** [reply]
https://mempool.space/tx/b0f4c3968fc6b660b05a6562c349cfea6fd24babc055163f67041adac4d765f8

**Reply by saubyk** [reply]
Can you try fee bumping one more time? This time use a higher fee rate, like 80 sats/vbyte or more.

also, can you see the pending transaction on the on-chain page of RTL?

**Reply by AndrewAB86** [reply]
lncli wallet bumpfee --sat_per_vbyte 100 b0f4c3968fc6b660b05a6562c349cfea6fd24babc055163f67041adac4d765f8:0
[lncli] rpc error: code = Unknown desc = the passed output does not belong to the wallet

I am getting this same error as before. No, it isn't showing on the onchain page as unconfirmed and the balance hasn't been added to my confirmed balance either. It's not showing anywhere on RTL but the tx is still on mempool at 1 sat fee

**Reply by saubyk** [reply]
try running this command: `lncli listunspent`. Do you see any unconfirmed transactions?
Also do you see any related record on the pending channels page? There must be a record on the `Pending Closing` section

You can try accelerating your transaction directly via mining pools:
https://www.viabtc.com/tools/txaccelerator

Worst case, you can also try the `Accelerate` button on the mempool.space (this will be expensive)
<img width="1140" alt="image" src="https://github.com/Ride-The-Lightning/RTL/assets/39208279/a20fad73-48c1-4ec9-bcc0-8e577a4931d7">

**Reply by AndrewAB86** [reply]
It was showing prior to this in the UTXO's. Now it is gone. My other UTXO's are still there which all show when I do lncli listunspent. It was always showing in pending closing prior to me having to restore onto a new hard drive, but hasn't shown since. After I did the restore there was no record of it anywhere on the node. I put this tx through nearly a year ago. It showed back up on my node in unfirmed sats a couple of weeks ago. I have tried the accelerate button in mempool but I am on the waiting list for that.

**Reply by AndrewAB86** [reply]
It has now just reappeared on my node. It is showing as unconfirmed balance and in the utxo list again. Still nothing in the pending closing though. Its not listed in any of the pending channels pages or closed channel pages
Looking on mempool I don't have RBF so does that mean i can't bump the fee through the terminal

**Reply by saubyk** [reply]
You can try `lncli wallet bumpfee...` again. I would recommend a fee rate higher than 50 sats/vb, given the prevailing mempool fee rates

Also explore: https://www.viabtc.com/tools/txaccelerator

**Reply by AndrewAB86** [reply]
I tried to do the bump fee again at 100 sats/vb but it still didnt change
guess I will have to go with viabtc

**Reply by AndrewAB86** [reply]
so viabtc worked and the tx has now gone through. It is sitting in UTXOs on RTL now. I have a few others sitting in there to. How do i get it out of this list

**Reply by saubyk** [reply]
Should be fine now I think, I see that sweep transaction now also has 5 confirmations. From the fee rate of the sweep, it appears to be the fee bump transaction, which may have been broadcast after the first one got confirmed.

You should now have all your UTXOs accounted for.

![image](https://github.com/Ride-The-Lightning/RTL/assets/39208279/5dbed7bd-8b80-4481-abac-2fd7ae0da4e8)

**Reply by AndrewAB86** [reply]
Got it all sorted now. Thanks for your help
